fe/src/main/java/org/apache/doris/task/StreamLoadTask.java

```diff
@@ -186,6 +186,7 @@ private void setOptionalFromRoutineLoadJob(RoutineLoadJob routineLoadJob) {
         partitions = routineLoadJob.getPartitions() == null ? null : Joiner.on(",").join(routineLoadJob.getPartitions());
         strictMode = routineLoadJob.isStrictMode();
         timezone = routineLoadJob.getTimezone();
+        timeout = (int) routineLoadJob.getMaxBatchIntervalS() * 2;
```

Comment:
Does FE send the timeout property to BE in query_options?

Reply:
Yes, previous we just use the Config.stream_load_default_timeout_second for the routine load task.